website/docs/Queries.md

```diff
@@ -389,18 +390,22 @@ An element is considered to be hidden from accessibility based on [`isHiddenFrom
 ```tsx
 render(<Text style={{ display: 'none' }}>I am hidden from accessibility</Text>);
 
-// Ignore hidden elements
+// Include hidden elements
 expect(
-  screen.queryByText('I am hidden from accessibility', { hidden: false })
+  screen.queryByText('I am hidden from accessibility', { includeHiddenElements: false })
 ).toBeFalsy();
 
 // Match hidden elements
-expect(screen.getByText('I am hidden from accessibility')).toBeTruthy(); // Defaults to hidden: true for now
+expect(screen.getByText('I am hidden from accessibility')).toBeTruthy(); // Defaults to includeHiddenElements: true for now
 expect(
-  screen.getByText('I am hidden from accessibility', { hidden: true })
+  screen.getByText('I am hidden from accessibility', { includeHiddenElements: true })
 ).toBeTruthy();
 ```
 
+### `hidden` option
+
+This is just an alias to the `includeHiddenElements` option. It only exists to match react-testing-library naming used in [byRole](https://testing-library.com/docs/queries/byrole/#hidden). Prefer the use of `includeHiddenElements` if possible as `includeHiddenElements: true` is clearer than `hidden: true`.
```

Comment:
Similar to `configure` comment, I would rather not give it a full option header but only mention it under   `includeHiddenElements` option docs.